### PR TITLE
Reword example to use terminology closer to NFC specs

### PR DESCRIPTION
**Pull request by kenchris**
***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/web-nfc/pull/337.html" title="Last updated on Sep 3, 2019, 2:51 PM UTC (c5ff904)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/337/e452164...kenchris:c5ff904.html" title="Last updated on Sep 3, 2019, 2:51 PM UTC (c5ff904)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -910,25 +910,33 @@ <h4>
...
+    </p>
+    <p>
+      As NDEF gives no guarantee on the ordering of records, using
+      an external type record with an <a>NDEF message</a> as payload,
+      can be useful for encapsulating related data.
     </p>
     <p>
-      Custom sub-record types start with a hyphen (`-`).
+      For external type records, local types are application specific.
+      Local types are exposed as a recordType, prefixed with "`-`".
```
Comment (zolkis) [inline index.html:934]: would add some more clarity if we'd say 

> , for instance prefixed with "`-`".

**Review by zolkis — APPROVED** [review]
Much more concise, thanks.
Eventually include the minor suggestion below.

File: index.html (hunk 1 above)
Comment (zolkis) [reply index.html:934]: Or, 

> In this example, local types are exposed as a recordType, prefixed with ...

**Comment by kenchris** [comment]
I will look at that when I add the parsing of local types. On phone now

**Pull request merged**